client/src/components/History/Content/Dataset/DatasetActions.vue

```diff
@@ -145,6 +165,9 @@ export default {
         onHighlight() {
             this.$emit("toggleHighlights");
         },
+        onInteractive(){
+            getEntryPoint(this.item.creating_job).then((data) => { window.open(data.data[0].target)})
```

Comment:
You could use the `UrlData` from `utils/url` here to avoid having a separate service module, other than that the implementation looks fine to me. Thank you for working on this.

Reply:
Yeah, I knew this was a hot-patch approach. Thank you for the insight how to do it right!